robottelo/constants/__init__.py

```diff
@@ -324,6 +324,11 @@
     'immediate': "Immediate",
 }
 
+MIRRORING_POLICIES = {
+    'additive': "Additive",
+    'mirror_complete': "Complete Mirroring",
+    'mirror_content_only': "Content Only",
```

Comment:
Use single quotes !

Reply:
I think he was probably matching the dictionaries on the lines below?

Reply:
Yeah, I was definitely style matching, doesn't make it right :)